### PR TITLE
Update geofence docker build process to new branch on datadir

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -64,6 +65,7 @@ This creates a ```georchestra/geoserver:geofence-15.12``` docker image:
 
 ```bash
 cd geoserver/geoserver-submodule/src
+rm -fr ../data/citewfs-1.1/workspaces/sf/sf/E*
```
Comment (fvanderbiest) [inline docker/README.md]: he ?

**Comment by fvanderbiest** [comment]
LGTM, but a bit of context is required for the `rm -fr ../data/citewfs-1.1/workspaces/sf/sf/E*` command.
Thanks

**Comment by Vampouille** [comment]
There is an encoding issue with this directory, so in order to build geoserver, we need to remove this folder :
`rm -fr ../data/citewfs-1.1/workspaces/sf/sf/E*`

Sorry for missing context :-)

**Comment by Vampouille** [comment]
This bug came from bad encoding of "é", in UTF-8 it should be encoded as xC3xA9 but current encoding is 'e' plus xCCx81

**Comment by Vampouille** [comment]
PR is created to fix folder encoding on upstream :
https://github.com/geoserver/geoserver/pull/1548

**Pull request merged**